pkgs/servers/home-assistant/custom-components/yandex-station/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  zeroconf,
+}:
+buildHomeAssistantComponent rec {
+  owner = "AlexxIT";
+  domain = "yandex_station";
+  version = "3.18.3";
+
+  src = fetchFromGitHub {
+    owner = "AlexxIT";
+    repo = "YandexStation";
+    rev = "v${version}";
+    hash = "sha256-l8DsL8g6K8/SjCIk7rjfQSk4iRsKBoGgzJpy7UhxQ7o=";
+  };
+  dependencies = [
+    zeroconf
+  ];
```

```suggestion

  dependencies = [
    zeroconf
  ];
  
  nativeCheckInputs = [
    home-assistant
    pytestCheckHook
  ] ++ (home-assistant.getPackages "stream" home-assistant.python.pkgs);
```

Some spacing, because things look a bit crammed. Also enables the test suite.